src/javaScript/common.js

```diff
@@ -0,0 +1,10 @@
+const hamburgerMenu = document.querySelector('#hamburger-menu')
+const nav = document.querySelector('#nav-index')
+
+function showElement (button, element) {
+  button.addEventListener('click', () => {
+    element.classList.toggle('flex')
+    element.classList.toggle('hidden')
+  })
+}
+showElement(hamburgerMenu, nav)
```

Comment:
Looking at this function it's confusing
ShowElement and i give it 2 parameters? Makes me think that you will showelement hamburgerMenu and showelement nav